data-reporting-service/observation-service/src/main/java/gov/cdc/etldatapipeline/observation/config/KafkaStreamsConfig.java

```diff
@@ -18,8 +19,7 @@ public class KafkaStreamsConfig {
 	@Value("${spring.kafka.bootstrap-servers}")
 	private String bootstrapServers;
 
-	@Value("${spring.kafka.streams.application-id}")
-	private String applicationId;
+	private String applicationId = UUID.randomUUID().toString();
```

Comment:
This is very helpful to manage multiple applications and rebuilds. Are there any concerns about this from the perspective of deployment or is it more of an internal id that is managed by Kafka?

Reply:
No there will not be any concerns due it being unique every time